cmd/loom/loom.go

```diff
@@ -1288,6 +1295,18 @@ func startPushGatewayMonitoring(cfg *config.PrometheusPushGatewayConfig, log *lo
 	}
 }
 
+func checkFileDescriptorLimit(min uint64) error {
+	var rlimit syscall.Rlimit
+	if err := syscall.Getrlimit(syscall.RLIMIT_NOFILE, &rlimit); err != nil {
+		return err
```

Comment:
Print out the error and return `nil` in this case, we don't want nodes refusing to start if this call fails due to some OS quirk.